.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn pretty-quick --staged
```

Comment:
pretty-quick and husky are both very fast and would only run on staged files so it would not be noticable. The benefit would be that the files of every commit will be automatically consistently formatted